components/DashboardPage.vue

```diff
@@ -7,6 +7,10 @@
             <ModIcon />
             My mods
           </nuxt-link>
+          <nuxt-link :to="'/dashboard/notifications'" class="tab last">
+            <NotificationsIcon />
```

Comment:
Wouldn't it be plausible to add a notifications link on the User drop-down on the navbar along side the dashboard?

Reply:
Add notifications to the dropdown; it is a good place for it